### PR TITLE
Feat/show expiry on position

### LINKED ISSUE #385: Show the expiry in the position and when creating an order
e.g. 00:00 every day

### PR DESCRIPTION
**Pull request by holzeis**
resolves #385

### CODE REVIEW

File: coordinator/src/node.rs [hunk 1]
```diff
@@ -66,7 +66,7 @@ impl Node {
         let margin_trader = margin_trader(trade_params);
         let margin_coordinator = margin_coordinator(trade_params);
 
-        let tomorrow = OffsetDateTime::now_utc().date() + Duration::days(1);
+        let tomorrow = OffsetDateTime::now_utc().date() + Duration::days(2);
```
Comment (1010Tom) [inline coordinator/src/node.rs:69]: Why +2 days?

Comment (holzeis) [reply coordinator/src/node.rs:69]: so it will be open for at least 24h, up to 47 hours and 59 mins.

**Review by 1010Tom — APPROVED** [review]
LGTM

**Pull request merged**